scipy/conftest.py

```diff
@@ -251,91 +266,79 @@ def _backends_kwargs_from_request(request, skip_or_xfail):
         if marker.kwargs.get('np_only'):
             kwargs['np_only'] = True
             kwargs['exceptions'] = marker.kwargs.get('exceptions', [])
-            kwargs['reason'] = marker.kwargs.get('reason', None)
+            kwargs['reason'] = marker.kwargs.get('reason')
```

Comment:
Does this make `reason` compulsory? If yes, I'd rather keep an option to not provide an explicit reason.

Reply:
The signature is `dict.get(key, default=None)`. I'll revert it to make it explicit as this doesn't seem to be common knowledge.